bin/load-runner.js

```diff
@@ -399,7 +397,15 @@ var l = new loop.MultiLoop({
       // if save output is true, create file containing actionStats in test output dir
       //       e.g. <curRuns>-<duration>.json
 
-      var dataToWrite = (dataJson !== null) ? JSON.stringify(dataJson, null, 2) : (dataRaw !== null) ? ('RAW DATA\n' + dataRaw) : 'no content returned from test script';
+      var dataToWrite;
```

Comment:
While the old version of this was confusing as hell with nested ternary operators, it *could have* led to one place where `dataToWrite` was assigned (if `const` was a thing back then). The new version is clearer, but now there are more places where the `dataToWrite` variable can be assigned from.

I think we could get the best of both worlds, by making that logic a function and returning the value to a `const dataToWrite` here. E.g.:

``` javascript
function chooseDataToWrite(json, raw) {
  if (json !== null) { //could it be not null but undefined?
    return JSON.stringify(json, null, 2);
  } else if (raw !== null) { //could it be not null but undefined?
    return ('RAW DATA\n' + raw);
  } else {
    return 'no content returned from test script';
  }
}

const dataToWrite = chooseDataToWrite(dataJson, dataRaw);
```

Would that be better? I think so, at least a little :)